scripts/exgfs_wave_init.sh

```diff
@@ -6,16 +6,10 @@
 # Script name:         exwave_init.sh
 # Script description:  Creates model definition files for WW3
 #
-# Author:   Jose-Henrique Alves Org: NCEP/EMC      Date: 2019-04-20
```

Comment:
Removal of author and script history log is inconsistent in this PR.  Should this be removed for all wave scripts?

Reply:
We probably should go over the doc-blocks of all scripts at some point in time and update them to reflect current maintainers.
If there is no objection, we can create an issue for this to work on in preparation for handoff closer to time.

Reply:
I mostly was curious why some and not others.  Totally fine to leave this for another day.